src/Request/Request.php

```diff
@@ -199,12 +187,75 @@ private function send($path, array $options = [])
         error_reporting(E_ALL ^ E_WARNING);
         $request = $this->getContainer()->get(HttpRequest::class, [$method, $uri, $headers, $body,]);
         error_reporting(E_ALL);
-        $response = $client->send($request);
+        $response = $this->sendWithRetry($client, $request);
 
         return $response;
     }
 
     /**
+     * Send the request using the Guzzle client. Retry the request if a server or network error occurs.
+     *
+     * @param Client $client
+     * @param HttpRequest $request
+     * @return array
+     * @throws \Pantheon\Terminus\Exceptions\TerminusException
+     */
+    private function sendWithRetry($client, $request)
+    {
+
+        $retry_interval = $this->getConfig()->get('http_retry_delay_ms', 100);
+        $retry_multiplier = $this->getConfig()->get('http_retry_backoff_multiplier', 2);
+        $retry_jitter = $this->getConfig()->get('http_retry_jitter_ms', 100);
+        $retry_max = $this->getConfig()->get('http_max_retries', 5);
```

Comment:
@ronan, This variable isn't used. we need something like `while ($tries < $retry_max)`, yes? 

And maybe some kind of message if the retry_max is reached.

Reply:
Or I just can't use the find command properly :)

Nevermind